adr/0006-dev-discussions.adoc

```diff
@@ -0,0 +1,122 @@
+        * Anyone can read
+        * Anyone with GitHub account can post/reply
+        * Moderation tools available to battle spam/noise
+    Disadvantages::::
+        * Notifications via email comes for *all* discussions, not possible to choose specific category
+        * User forums gets intermixed with Developer/Design discussions
+
+Primarily use issues/PR's::
+    Advantages::::
+        * Conversation directly linked to code changes
+    Disadvantages::::
+        * Not possible to just monitor the overall design discussions as all mixed up with anything else
+        * Notification filtering is quite limited
+
+Primarily use Zulip Chat::
+    Advantages::::
+        * Real-time communication
+    Disadvantages::::
+        * Only for those who have time/privilege to follow along
+
+Use private chat/mail::
+    Advantages::::
+        * Direct and focused private communication (sometimes useful)
+    Disadvantages::::
+        * Not great for open governance unless culture to comment/report on conclusion somewhere more public
+        * Cannot be located/found when trying to search for answers/why something was done
+
+Status quo/Do nothing::
+    Advantages::::
+        * Nothing need to change
+    Disadvanteges::::
+        * All of the above disadvantages
+        * No clear place to go for design discussions
+
+Seperate quarkusio/quarkus-dev repository with discussions::
+    Advantages::::
+        * Clear place to go for design discussions
+        * Easy to find and follow discussions
+        * Can filter/focus separately on user versus developer discussions
+        * Can use e-mail, at least for commenting
+        * ADR can be moved there
+        * Future call minutes can be moved there
+        * Does not break existing user conversations
+    Disadvantages::::
+        * Need to move discussions from other places to this new place
+        * Need to retire quarkus-dev mailing list
+        * Users conversation is in the repo where code is, developer conversation is in a separate repo
+
+== Decision
+
+- Introduce a new GitHub repository called `quarkusio/quarkus-dev`.
+  * Have a README with links to discussions, ADRs, call minutes etc.
+  * Enable discussions in the repository
+  * Use @quarkus-bot for tagging
+- Move existing quarkusio/quarkus ADRs to the new repository
+  * keep `adr` directory in the main repository with a README pointing to the new location
+- Retire `quarkus-dev` mailing list
+  * Add a footer to all mails with a message to go to the new repository
+  * Change the name of the mailing list to `quarkus-dev-retired`
+  * Post last e-mail to the list with a message to go to the new repository
+  * Eventually (6-12 months?) enable moderation 
+    ** not doing it on day one as it will take a while for 1000+ members to get the message
+- Encourage moving Zulip chat introducing/tracking ideas to discussions
+  * Encourage summarizing chat outcomes on discussions/issues
+
```

Comment:
- All posts we before did to quarkus-dev Mailing list should go to a category on the quarkus-dev/discussion
    * (pre)release announcement
    * ideas, suggestions, etc.